src/components/surfitProfile/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import logo from 'assets/icon_logo.svg';
+import profilePic from 'assets/mockup_profile.svg';
+import { React } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+const NavContainer = styled.ul`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  list-style: none;
+  width: 34.3rem;
+  height: 3.2rem;
+  font-size: 1.4rem;
+  color: ${theme.colors.surfit_grey_3};
+  margin: 5.2rem 0 auto;
+`;
+
+const Img = styled.img`
+  width: 3.2rem;
+  height: 3.2rem;
+`;
+
+const Logo = styled(Img)`
+  border: 1.5px solid white;
+  border-radius: 10rem;
```

Comment:
요고 동그랗게 하기 위한거라면 50%주면 됨니다!
```suggestion
  border-radius: 50%;
```